Dockerfile

```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
 
 # Build the binary
-FROM --platform=${BUILDPLATFORM} docker.io/golang:1.23.10 AS builder
+FROM --platform=${BUILDPLATFORM} docker.io/golang:1.24 AS builder
```

Comment:
Can we please set the full Go version to make sure we're in line with the build images?

Reply:
I would be okay with that as well.